src/diffusers/loaders/lora_pipeline.py

```diff
@@ -277,6 +281,7 @@ def lora_state_dict(
                 The subfolder location of a model file within a larger model repository on the Hub or locally.
             weight_name (`str`, *optional*, defaults to None):
                 Name of the serialized state dict file.
+            return_lora_metadata: TODO
```

Comment:
To not break backwards-compatibility. In case the state dict doesn't contain any metadata it will be None despite `return_lora_metadata=True`.